anago

```diff
@@ -156,14 +156,13 @@ common::cleanexit () {
 
   if [[ -d $WORKDIR ]]; then
     logecho -n "Copying $LOGFILE to $WORKDIR: "
-    logrun -s cp -f $LOGFILE $WORKDIR
+    logrun -s cp -f $LOGFILE* $WORKDIR
```

Comment:
For an archive operation, I want to include all log files and it turns out it's not a bad idea in the general use case as well.  L153 should get pulled into that condition as well.  Thanks.